frontend/src/App.js

```diff
@@ -1,15 +1,19 @@
 import React, { useEffect } from "react";
 import { Provider } from "react-redux";
+import ReactGA from "react-ga";
 import store from "./redux";
 import Pages from "./pages";
 import { VisaStatusNotification } from "./components";
-import { useInitialDataFetch } from "./hooks";
+import { useInitialDataFetch, useScript } from "./hooks";
 import "./assets/styles/index.less";
 
 let RENDER_TIME = 0; // for debug purpose delete before production
 
 function App() {
+    ReactGA.initialize("UA-102409527-2");
+    ReactGA.pageview(window.location.pathname + window.location.search);
     useInitialDataFetch();
+    useScript("https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js");
```

Comment:
~~We could potentially go extreme and write a `useInitialization` to group all the hooks into one hook, just saying~~